plugin.info.txt

```diff
@@ -1,7 +1,7 @@
 base   singlesearchresult
 author Matthias Schulte
```

Comment:
no clue if adding my name here is appropriate or technically a good idea (same for email)